release.nix

```diff
@@ -27,6 +28,7 @@ let
     cardano-sl-explorer-static = [ "x86_64-linux" ];
     cardano-report-server-static = [ "x86_64-linux" ];
   };
-in (mergeAttrsMap (dconfig: { ${dconfig } = (withDconfig dconfig).mapTestOn platforms; }) dconfigs)
-   // ((withDconfig null).mapTestOn { stack2nix = supportedSystems; })
+in (mergeAttrsMap (dconfig: { ${dconfig } = (withDconfig dconfig false).mapTestOn platforms; }) dconfigs)
+   // ((withDconfig null false).mapTestOn { stack2nix = supportedSystems; })
    // (rlib.mapTestOn { purescript = supportedSystems; })
+   // (mergeAttrsMap (dconfig: { "${dconfig}-profiled" = (withDconfig dconfig true).mapTestOn platforms; }) dconfigs)
```

Comment:
As discussed this needs to be rebased on top of `cardano-sl-1.0-genesis-json2` branch and then we build profiled build only of `cardano-sl-core` on Linux.